fastlane/lib/fastlane/actions/copy_artifacts.rb

```diff
@@ -25,7 +25,7 @@ def self.run(params)
           missing = artifacts.reject { |a| File.exist?(a) }
           UI.user_error!("Not all files were present in copy artifacts. Missing #{missing.join(', ')}") unless missing.empty?
         else
-          # If we don't fail on non-existent files, don't try to copy non-existent files
+          # If we don't fail on nonexistent files, don't try to copy nonexistent files
```

Comment:
This unhyphenated form is already used a bit in the repo (and I vastly prefer it over the hyphenated form).